### PR TITLE
Timer Race

### PR DESCRIPTION
**Pull request by Maithem**
## Overview
Fixed a race that caused a NullPointerException when trying to
get a timer in sendMessageAndGetCompletable.

Why should this be merged: Fixes a race condition that can cause a NullPointerException when calling sendMessageAndGetCompletable

## Checklist (Definition of Done):

- [x] There are no TODOs left in the code
- [x] [Coding conventions](https://github.com/CorfuDB/CorfuDB/wiki/Corfu-Style-Guidelines) (e.g. for logging, unit tests) have been followed
- [x] Change is covered by automated tests
- [x] Public API has Javadoc

### CODE REVIEW

File: runtime/src/main/java/org/corfudb/runtime/clients/NettyClientRouter.java [hunk 1]
```diff
@@ -167,6 +167,12 @@ public NettyClientRouter(@Nonnull NodeLocator node,
         this.parameters = parameters;
         this.eventLoopGroup = eventLoopGroup;
 
+        // Set timer mapping
+        for (CorfuMsgType type : CorfuMsgType.values()) {
+            timerNameCache.put(type,
```
Comment (xnull) [inline runtime/src/main/java/org/corfudb/runtime/clients/NettyClientRouter.java]: timerNameCache could be an ImmutableMap from guava to explicitly show that the map is immutable, that would be helpful for the readers

Comment (Maithem) [reply runtime/src/main/java/org/corfudb/runtime/clients/NettyClientRouter.java]: Done.

File: runtime/src/main/java/org/corfudb/runtime/clients/NettyClientRouter.java [hunk 2]
```diff
@@ -49,6 +50,7 @@
 import java.util.List;
 import java.util.Map;
 import java.util.NoSuchElementException;
+import java.util.UUID;
```
Comment (corfu-repo) [inline runtime/src/main/java/org/corfudb/runtime/clients/NettyClientRouter.java]: MINOR Remove this unused import 'java.util.UUID'. [rule](https://sonarcloud.io/coding_rules#rule_key=squid%3AUselessImportCheck)

**Comment by corfu-repo** [comment]
SonarQube analysis reported 2 issues
* ![MINOR][MINOR] 2 minor

Watch the comments in this conversation to review them.

#### 1 extra issue

Note: The following issues were found on lines that were not modified in the pull request. Because these issues can't be reported as line comments, they are summarized here:

1. ![MINOR][MINOR] [NettyClientRouter.java#L49](https://github.com/CorfuDB/CorfuDB/blob/79f7b9ac01dbeb47834de57ed9d17153e7da68b8/runtime/src/main/java/org/corfudb/runtime/clients/NettyClientRouter.java#L49): Remove this unused import 'java.util.HashMap'. [rule](https://sonarcloud.io/coding_rules#rule_key=squid%3AUselessImportCheck)

[MINOR]: https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR'

**Pull request merged**